lib/chef_apply/ui/terminal.rb

```diff
@@ -49,14 +50,19 @@ def render_parallel_jobs(header, jobs)
           # @option options [Hash] :style
           #   keys :top :middle and :bottom can contain Strings that are used to
           #   indent the spinners. Ignored if message is blank
-          multispinner = get_multispinner.new("[:spinner] #{header}", output: @location, hide_cursor: true, style: indent_style)
+          multispinner = get_multispinner.new("[:spinner] #{header}", output: @location,
+                                                                      hide_cursor: true, style: indent_style)
```

Comment:
If wrapping args should they all lineup? or at least put each option on it's own line?